test/test_dataDict.py

```diff
@@ -0,0 +1,127 @@
+import gw_eccentricity
+from gw_eccentricity import load_data
+from gw_eccentricity import measure_eccentricity
+from gw_eccentricity.utils import time_deriv_4thOrder
+import numpy as np
+
+
+def test_dataDict():
+    """
+    Test that the interface works for dataDict with all allowed
+    combination of keys in dataDict.
+    """
+    # Load test waveform
+    lal_kwargs = {"approximant": "EccentricTD",
+                  "q": 1.0,
+                  "chi1": [0.0, 0.0, 0.0],
+                  "chi2": [0.0, 0.0, 0.0],
+                  "Momega0": 0.01,
+                  "ecc": 0.1,
+                  "mean_ano": 0,
+                  "include_zero_ecc": True}
+    dataDict = load_data.load_waveform(**lal_kwargs)
+
+    # get amplm, phaselm, omegalm
+    phaselm = {}
+    amplm = {}
+    omegalm = {}
+    dt = dataDict["t"][1] - dataDict["t"][0]
```

Comment:
Add a sanity check that the time array is uniform. Good to add checks like this in multiple places, so that when we eventually switch to nonuniform time arrays, we know which places changes need to be made. 

In general, important to think about what your code is assuming, and add sanity checks -- I always try to do the exercise of "How can I break my code?".

Reply:
Done